doc/src/pg_developer.rst

```diff
@@ -48,6 +52,9 @@ Some source files and classes do not have a corresponding input script
 command, e.g. ``src/force.cpp`` and the :cpp:class:`LAMMPS_NS::Force`
 class.  They are discussed in the next section.
 
+A small number of C++ classes are implemented with only a ``.h`` file.
```

Comment:
The example given here is not correct. While fmtlib can be used and compiled as an inline library, we don't use it like this. The implementation defined in the headers is in `src/fmtlib_format.cpp` and `src/fmtlib_os.cpp`. Examples for a header-only classes would be `src/pointers.h` or `src/my_page.h`

Reply:
Thank you for the correction.